config/routes.rb

```diff
@@ -50,7 +50,7 @@
     end
 
     namespace :v2 do
-      resources :people, only: %i[index]
+      resources :people, only: %i[index create]
```

Comment:
```suggestion
      resources :people
```

Reply:
at the moment we only have the index and create actions, I am missing something?